configure.ac

```diff
@@ -497,6 +497,9 @@ AS_IF([test "x$with_julia" != xno ],[
   AS_IF([ test $? != 0 ], [AC_MSG_ERROR([failed to obtain JULIA_LIBS from julia-config.jl])])
   JULIA_LIBS=${JULIA_LIBS//\'/}
   AC_MSG_RESULT([${JULIA_LIBS}])
+  AC_MSG_NOTICE([installing GAPTypes.jl])
+  ${JULIA} -e 'import Pkg; Pkg.add("GAPTypes")'
+  AS_IF([ test $? != 0 ], [AC_MSG_ERROR([failed to install GAPTypes.jl])])
```

Comment:
Did you verify that this works as intended? (E.g. by changing the previous line to use `Pkg.add("foobarquux")` to force an error?

Reply:
Yep.